ui/pages/api/lectures/[id].tsx

```diff
@@ -1,16 +1,6 @@
-import { Client } from "pg";
+import { findLectureById } from "../../../api/psql_client";
 
 export default async function handler(req, res) {
   const { id } = req.query;
-  const client = new Client({
-    user: "cat",
-    password: "cat",
-  });
-  await client.connect();
-
-  const result = await client.query(
-    "SELECT * FROM cds_lecture WHERE lecture_id = " + id
-  );
-  await client.end();
-  res.status(200).json(result.rows[0]);
+  res.status(200).json(findLectureById(id));
```

Comment:
would be nice to put error handling, or we don't need it?

Reply:
Added